src/domain/community/community/community.service.ts

```diff
@@ -423,36 +426,70 @@ export class CommunityService {
     return room;
   }
 
-  async sendUpdateMessageToCommunity(
+  async sendMessageToCommunityUpdatesRoom(
     community: ICommunity,
     email: string,
-    msgData: CommunitySendMessageInput
+    messageData: CommunitySendMessageInput
   ): Promise<string> {
     await this.communicationService.ensureUserHasAccesToCommunityMessaging(
       community.communicationGroupID,
       community.updatesRoomID,
       email
     );
-    return await this.communicationService.sendMsgCommunity({
+    return await this.communicationService.sendMessageToCommunityRoom({
       sendingUserEmail: email,
-      message: msgData.message,
+      message: messageData.message,
       roomID: community.updatesRoomID,
     });
   }
 
-  async sendDiscussionMessageToCommunity(
+  async sendMessageToCommunityDiscussions(
```

Comment:
Should be sendMessageToCommunityDiscussions**Room** as in sendMessageToCommunityUpdatesRoom for consistency

Reply:
Addressed in 896940843be935910f4036fba30e554b1e1411ad - removed Room from all methods